### PR TITLE
Updating quickstart guide for PPRL approach

### LINKED ISSUE #1331: Update documentation to reflect architecture changes
Update the Piipan documentation after recent changes to implement deidentification
- newcomers to project should understand that Piipan avoids centralized collection of participant records
- architecture should be updated to show how this will be achieved
- review older documentation to ensure we're not using language that reflects direct collection / storage of participant PII.

Also:
- [ ] Clarify the degree of cloud platform agnostic approach we are aiming for (12-factor apps/microservices/PostgreSQL/general architecture patterns, but not to avoid APIM, Front Door, Event Grid)

### PR DESCRIPTION
**Pull request by ryanhofdotgov**
Closes #1331

### CODE REVIEW

File: docs/quick-start-guide-states.md [hunk 1]
```diff
@@ -1,79 +1,64 @@
...
 
-APIs are in Alpha stage and under active development. APIs are available for state testing now.
+These actions correspond to 2 web service APIs:
 
-## APIs Overview
+1. [Bulk Upload API](./openapi/generated/bulk-api/openapi.md)
+1. [Duplicate Participation API](./openapi/generated/duplicate-participation-api/openapi.md)
 
-In order to participate, states will need to:
+Each API has one or more RPC or REST operations and uses JSON in the operation request and/or response bodies. All operations must be made over HTTPS are authenticated by an API key. Each state will be issued a key for the Bulk Upload API and a separate key for the Duplicate Participation API.
```
Comment (kingcomma) [inline docs/quick-start-guide-states.md:17]:
```suggestion
Each API has one or more RPC or REST operations and uses JSON in the operation request and/or response bodies. All operations must be made over HTTPS and authenticated by an API key. Each state will be issued a key for the Bulk Upload API and a separate key for the Duplicate Participation API.
```

File: docs/quick-start-guide-states.md [hunk 2]
```diff
@@ -1,79 +1,64 @@
...
+- [Personal Identifiable Information (PII) de-identification](./pprl.md)
+- [Duplicate Participation API](./openapi/generated/duplicate-participation-api/openapi.md)
 
-These three elements translate to two main API calls that states will integrate into their existing eligibility systems and workflows:
+## Notional, high-level integration steps
+1. Understand the NAC material in the [system overview](https://github.com/18F/piipan#overview) and our [introduction to our Privacy-Preserving Record Linkage approach](https://github.com/18F/piipan/blob/dev/docs/pprl-plain.md).
+1. Export participant data in plain text from your eligibility system to CSV.
+1. Transform the plain text CSV to the Bulk Upload CSV format, in accordance with the PII de-identification specification.
+1. Integrate with the Bulk Upload API to submit the CSV to the NAC using the `/upload` operation.
+1. Integrate with the Duplicate Participation API to conduct searches, de-identifying applicant PII, before making the `/find_matches` call.
```
Comment (kingcomma) [inline docs/quick-start-guide-states.md:30]:
```suggestion
1. Integrate with the Duplicate Participation API to conduct searches, de-identifying applicant PII before making the `/find_matches` call.
```

**Review by kingcomma — APPROVED** [review]
👍 Looks great — just a couple small copy edit suggestions.

**Comment by ryanhofdotgov** [comment]
@kingcomma Thanks for the edits!

**Pull request merged**